src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.spec.ts

```diff
@@ -67,6 +72,11 @@ describe(commands.TEAMS_REPORT_DEVICEUSAGEDISTRIBUTIONUSERCOUNTS, () => {
     assert.notEqual(actual, true);
   });
 
+  it('fails validation on invalid output', () => {
```

Comment:
I don't think we need this test here since we're not introducing a new output type and json and text are already covered in the base class.

Reply:
Done. This is fixed